src/applications/verify-your-enrollment/tests/helpers/translateDateIntoMonthDayYearFormat.unit.spec.js

```diff
@@ -21,6 +21,9 @@ describe('translateDateIntoMonthDayYearFormat', () => {
     expect(translateDateIntoMonthDayYearFormat('2023-01-25')).to.equal(
       'January 25th, 2023',
     );
+    expect(translateDateIntoMonthDayYearFormat('1923-01-25', true)).to.equal(
+      'January 25th, 2023',
```

Comment:
Is this meant to have 1923 in the arg, but 2023 in the expected output? Just making sure this isn't an unintentional mismatch. I see the tests have passed on the branch, just being cautious.

Reply:
Hi @CBonade 
It is. This is one of the things we are checking for. the data base that sends us the information is so old that it sends dates as if the date is in the 1900s when it really should be in the 2000s. Meaning 1923, would equal 2023 after the date is ran through the function.